src/main/config/config.c

```diff
@@ -384,9 +384,9 @@ static void resetConf(void)
 
     // global settings
     masterConfig.current_profile_index = 0;     // default profile
-    masterConfig.gyro_cmpf_factor = 600;        // default MWC
+    masterConfig.gyro_cmpf_factor = 400;        // default MWC
     masterConfig.gyro_cmpfm_factor = 250;       // default MWC
-    masterConfig.gyro_lpf = 42;                 // supported by all gyro drivers now. In case of ST gyro, will default to 32Hz instead
+    masterConfig.gyro_lpf = 88;                 // supported by all gyro drivers now. In case of ST gyro, will default to 32Hz instead
```

Comment:
is changing the two default values above strictly necessary?  I'm cautious of balancing between making cleanflight awesome for new users and not forcing people to re-tune after an upgrade.  Same goes for default overclocking and looptime below.